nailgun/entities.py

```diff
@@ -1487,7 +1506,7 @@ def __init__(self, server_config=None, **kwargs):
                 OperatingSystem,
                 null=True,
             ),
-            'parent': entity_fields.OneToOneField(HostGroup, null=True),
+            'ancestry': entity_fields.StringField(null=True),
```

Comment:
It seems more StringField for me as, for example, "ancestry":"131" doesn't seem as a hash for me.